src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -195,7 +195,7 @@ public Path getJavaAppEngineSdkPath() {
     return getSdkPath().resolve(JAVA_APPENGINE_SDK_PATH);
   }
 
-  private Path getJavaToolsJar() {
+  public Path getJavaToolsJar() {
```

Comment:
I would like to worry about the case where the SDK is split into different directories later, when we have to.